CAIPs/caip-25.md

```diff
@@ -106,22 +123,21 @@ Each object should be keyed to the scope of a `sessionScopes` member to which it
 All properties of each object in `scopedProperties` MUST be interpreted by the respondent as proposals or declarations rather than as requirements.
 In addition to making additional properties of or metadata about the corresponding `sessionScopes` member explicit, they can also annotate, support, or extend the negotiation of scope proposals (e.g., providing connection information about unfamiliar scopes or which accounts to expose to each).
 
+When there is an active session between the wallet and the caller for which the wallet assigned a `sessionId` in it's initial response, a `sessionId` parameter, it's value  the same entropic identifier returned by the wallet, should be added to the `wallet_createSession` request to update the associated session.
```

Comment:
```suggestion
When there is an active session between the wallet and the caller for which the wallet assigned a `sessionId` in it's initial response, that same `sessionId` should be added to the `wallet_createSession` request to update the associated session.
```
Wording was a little confusing, we've already identified `sessionId` as being a CAIP-171 identifier, seems redundant to state again.

Reply:
done here:  https://github.com/ChainAgnostic/CAIPs/pull/285/commits/71b77444c778d0f5a98885d14529c26ff8d9de22